sycl/test/basic_tests/accessor/accessor.cpp

```diff
@@ -410,15 +480,83 @@ int main() {
     sycl::accessor<int, 0, sycl::access::mode::read,
                    sycl::access::target::host_buffer>
         acc4(buf1);
+#ifndef simplification_test
     sycl::accessor<int, 1, sycl::access::mode::read,
                    sycl::access::target::host_buffer>
         acc5(buf2);
     sycl::accessor<int, 1, sycl::access::mode::read,
                    sycl::access::target::host_buffer>
         acc6(buf3, sycl::range<1>(1));
+#else
+    sycl::host_accessor acc5(buf2, sycl::read_only);
+    sycl::host_accessor acc6(buf3, sycl::range<1>(1), sycl::read_only);
+#endif
 
     assert(acc4 == 2);
     assert(acc5[0] == 4);
     assert(acc6[0] == 6);
   }
+
+#ifdef simplification_test
```

Comment:
I think these cases are a worthwhile addition for the non-simplified test as well, in the same manner that all the test cases above are done for both versions.

Reply:
Done